frontend/src/metabase/entities/questions.js

```diff
@@ -34,12 +34,24 @@ const Questions = createEntity({
         ),
       ),
 
-    setCollection: ({ id, model }, collection, opts) =>
-      Questions.actions.update(
-        { id },
-        { collection_id: canonicalCollectionId(collection && collection.id) },
-        undo(opts, model === "dataset" ? "model" : "question", "moved"),
-      ),
+    setCollection: ({ id, model }, collection, opts) => {
```

Comment:
@daltojohnso FYI I changed from always reloading the collection to the a similar approach used in [#21217](https://github.com/metabase/metabase/pull/21217/files#diff-78066c012a6bedfa36b098d60dd0a26afda8f22318437a85e46dd3f2eed7864a), so we would only refresh Collections when necessary.